h2/src/docsrc/help/help.csv

```diff
@@ -5878,7 +5878,7 @@ H2VERSION()
 
 "Functions (JSON)","JSON_OBJECT","
 JSON_OBJECT(
-[{[KEY] string VALUE expression} | {string : expression} [,...]]
+[{[KEY] string VALUE expression} [,...] | {string : expression} [,...]]
```

Comment:
Please, change it to `[{{[KEY] string VALUE expression} | {string : expression}} [,...]]` instead. Mixed syntax is perfectly valid in both SQL:2016 and H2's implementation.

Reply:
This doesn't seem to work:

```sql
select json_object('a' value a, b)
from (values (1, 2)) t(a, b)
```

I'm getting:

```
Syntax error in SQL statement "SELECT JSON_OBJECT('a' VALUE A, B)[*]
FROM (VALUES (1, 2)) T(A, B)"; expected "(, ., [, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, VALUE, :"; SQL statement:
select json_object('a' value a, b)
from (values (1, 2)) t(a, b) [42001-200]
```

Reply:
My bad. Yes, it works

Reply:
Additional commit added: https://github.com/h2database/h2database/pull/2495/commits/63d67b6163403847f064673e3ecc92cdead97b31